renderer.php

```diff
@@ -47,7 +47,8 @@ public function formulation_and_controls(question_attempt $qa,
             'isPreview' => false,
             'title' => get_string('celltitle', 'qtype_crossword'),
             'orientation' => $orientationvalue,
-            'readonly' => false
+            'readonly' => false,
+            'cellSize' => [60, 40]
```

Comment:
We pass in cell size here, but then in lots of other places, we hard-code the magic numbers 60 and 40, so this is just a mess.

I could try to work out what is acutally going on, but acutally I won't. If it is only possible to understand the code after an hour of hard thought, the code is not good enough, and should first be rewritten to be much clearer, and then explained.